README.md

```diff
@@ -122,14 +124,27 @@ To install `Ubuntu Make`, run the following command:
 
 	   
 ## 2. Installing the resource aggregator for ODIM and GRF plugin
-This section provides a step-by-step procedure for deploying the resource aggregator for ODIM (odimra) and GRF plugin.
+This section provides a step-by-step procedure for deploying ODIMRA and GRF plugin.
 
   
   **NOTE:**
   - All configuration parameters are set to default values in the configuration files for odimra and GRF plugin. 
   - The following ports are used for deploying odimra and GRF plugin:
     45000, 45001, 45101-45110, 9092, 9082, 6380, 6379, 8500, 8300, 8302, 8301, 8600
     Ensure that the above ports are not in use.
+  - The following users are created and added to group ids automatically when the certificates are generated during deployment. 
+  
+    |User Id| Group Id|
+	-----|---------|
+	|`odimra`|1234 |
+	|`plugin`|1235 |
+	
+
+     `odimra` is created on both the VM and the container for the resource aggregator.
```

Comment:
VM ? We are developing this for k8s which is primarily deployed on BM. Docker images could run on either. So remove references to VM. What is the Group Id ?